.github/workflows/ci.yml

```
@@ -76,18 +76,26 @@ jobs:
 
 
   build-and-test-python-wheels:
-    name: Build and test Python wheels on ${{ matrix.os }}
+    name: Build and test wheels on ${{matrix.arch}} for ${{ matrix.os }}
```

I think the other way around is more correct?

```suggestion
    name: Build and test wheels on ${{ matrix.os }} for ${{matrix.arch}} 
```